src/main.rs

```diff
@@ -17,6 +17,30 @@ mod utils;
 
 const EPOCH_LENGTH: u64 = 43200;
 
+fn print_table(table: &Table, reward_sum: f64, liquid_balance_sum: f64, price: f32) {
+    let mut table = table.clone();
+
+    table.add_row(Row::new(vec![
+        Cell::new(format!("${:.2}", (reward_sum as f32)).as_str())
+            .with_hspan(3)
```

Comment:
```suggestion
            .with_hspan(2)
```

It's easier to read if the reward sum is placed right under the reward